include/podio/ROOTDataSource.h

```diff
@@ -0,0 +1,178 @@
+#ifndef PODIO_DATASOURCE_H__
+#define PODIO_DATASOURCE_H__
+
+// STL
+#include <string>
+#include <vector>
+
+// ROOT
+#include <ROOT/RDataFrame.hxx>
+#include <ROOT/RDataSource.hxx>
```

Comment:
STL should go all the way to the bottom, ROOT above them, such that the podio headers are first.

Reply:
done